test/Conversion/TppToXsmm/tpp-to-xsmm-quarternary.mlir

```diff
@@ -1,4 +1,5 @@
 //RUN: tpp-opt %s --combine-tpp --convert-tpp-to-xsmm | FileCheck %s
+// XFAIL:*
```

Comment:
Nope, we can't XFAIL tests without a proper review. @KavithaTipturMadhu @adam-smnk

Reply:
Here to be honest I don't understand the test and now combine-tpp runs at tensor, so now sure what to do with this..

Reply:
The test might need to be rewritten but we still need to check that now `tpp.brgemm` with VNNI input gets lowered to `xsmm.quarternary`.

Reply:
Ok, I actually reenabled it few commits later. If you don't mind I will do this after we fix also the quaternary stuff, as I am actually changing it.

Reply:
It is unclear what this tests what to do. If it checks only the lowering from tpp to xsmm for the quaternary, which is already covered or the combined pass. The conversion should be tested in isolation and it is exercised in `tpp-to-xsmm-quarternary.mlir`. To me we can delete it.

Reply:
Asserting that the test is covered elsewhere and removing the test is also a valid outcome. So is splitting the test into smaller steps (like we did previously) while making sure we don't lose test coverage.

Reply:
Test is already covered. See new exmaples for the fusion pass and we have the conversion in`tpp-to-xsmm-quaternary.mlir`.